lib/lita-lean_tc.rb

```diff
@@ -62,6 +66,15 @@ def build_finished(payload)
           create_confirmed
         end
       end
+      # Returns a list of all cards on the Feature request board
+      #
+      def list_feature_request(response)
```

Comment:
My bad, I now see the two are different.

What if you move this logic into a new method in the `FeatureRequests` class? You could call it `all_feature_requests`, then rename the `to_msg` method to be `old_feature_requests` or something like that. My main motivation for this is to have all the feature request logic in the one place.